openpype/hosts/maya/plugins/publish/extract_fbx_animation.py

```diff
@@ -42,6 +42,12 @@ def process(self, instance):
         # Export from the rig's namespace so that the exported
         # FBX does not include the namespace but preserves the node
         # names as existing in the rig workfile
+        if not out_members:
+            self.log.debug(
+                "Top group of animated skeleton not found.."
+                "skipping extraction")
+            return
```

Comment:
```suggestion
        if not out_members:
            self.log.debug(
                "Top group of animated skeleton not found. "
                "Skipping FBX animation extraction.")
            return
```

Reply:
This extractor is only active when users enable the fbx animation collector. Maybe we should improve the debug message a little bit to allow users understand why the fbx export is skipping.(e.g.Top group of animation skeleton is not found in {set}, Skipping Fbx animation extraction)